toonz/sources/translations/german/tnzcore.ts

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="de_DE">
+<context>
+    <name>QObject</name>
+    <message>
+        <source>colors</source>
+        <translation>Farben</translation>
+    </message>
+    <message>
+        <source>Unidentified Action</source>
+        <translation>Unbekannte Aktion</translation>
+    </message>
+</context>
+<context>
+    <name>TCenterLineStrokeStyle</name>
+    <message>
+        <source>Constant</source>
+        <translation>Konstante</translation>
+    </message>
+    <message>
+        <source>Thickness</source>
+        <translation>Dicke</translation>
+    </message>
+</context>
+<context>
+    <name>TRasterImagePatternStrokeStyle</name>
+    <message>
+        <source>Distance</source>
+        <translation>Abstand</translation>
+    </message>
+    <message>
+        <source>Rotation</source>
+        <translation>Rotation</translation>
+    </message>
+</context>
+<context>
+    <name>TVectorImagePatternStrokeStyle</name>
+    <message>
+        <source>Distance</source>
+        <translation>Abstand</translation>
```

Comment:
You're right, I just had the feeling that "Entfernung" would be better here - today I'm not so sure anymore, so it's just a matter of opinion. ;-)